config/global.ini.php

```diff
@@ -719,6 +719,10 @@
 ; If set to 0 it also disables the "sent plugin update emails" feature in general and the related setting in the UI.
 enable_update_communication = 1
 
+; This option defines the protocols Matomo's Http class is allowed to open.
+; If you may need to download GeoIP updates or other stuff using other protocols like ftp you may need to extend this list.
+allowed_protocols = 'http,https'
```

Comment:
Do you think it might be better to name this `allowed_outgoing_protocols` or similar? To avoid confusion about what it refers to.

Reply:
updated the name